src/test/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/InputRedundantModifierFinalInEnumMethodsInFinalClass.java

```diff
@@ -0,0 +1,16 @@
+package com.puppycrawl.tools.checkstyle.checks.modifier.redundantmodifier;
+
+public final class InputRedundantModifierFinalInEnumMethodsInFinalClass {
+    public enum TestEnum {
+        VALUE {
+           @Override public void someMethodToOverride() {
+                // do it differently
+            }
+         };
+
+        public void someMethodToOverride() { }
+        public static final void someStaticMethod() { }
```

Comment:
as this Check does not have properties, please add trailing comment "// violation" to clearly define place of violation.

Reply:
Done.